src/modules/esl-base-element/README.md

```diff
@@ -1,28 +1,57 @@
 # [ESL](../../../) Base Element
 
-Version: *1.0.0*
+Version: *1.1.0*
 
 Authors: *Alexey Stsefanovich (ala'n)*
 
 <a name="intro"></a>
 
-Provides the ESLBaseElement - a base class for custom element declaration, and a set of common decorators.
+Provides the core to create custom elements and ts decorators to simplify components creation.
```

Comment:
Provides the core to create custom elements and ts decorators to simplify components developing.